src/history/config.go

```diff
@@ -197,6 +205,7 @@ func init() {
 	historyFile = helper.Env("HISTORY_FILE", "/etc/nginx/logs/history.json")
 	healthCheckFile = helper.Env("HEALTH_CHECK_FILE", "/etc/nginx/logs/health.json")
 	stateFile = helper.Env("STATE_FILE", "/etc/nginx/logs/state.json")
+	tagSecret = helper.Env("TAG_SECRET", "")
```

Comment:
Yeah, I'll do that. Though, could you explain why we need to add something to the timestamp and repo URL to hash it? Some security concerns?

Reply:
There is not much need, because everything is internal and no other package can change the history map's state. Generally for public facing requests you add a secret, so that MitM/replay attacks cannot happen. You can also ignore it.